### PR TITLE
Add Databricks to the Cloud Data Ingestion documentation

### PR DESCRIPTION
**Pull request by andrewcarlberg**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [x] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [x] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [x] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 1]
```diff
@@ -241,6 +241,62 @@ If you have network policies in place, you must give Braze network access to you
...
+- User identifier columns. Your table may contain one or more user identifier columns. Each row should only contain one identifier (either `external_id`, the combination of `alias_name` and `alias_label`, or `braze_id`). A source table may have columns for one, two, or all three identifier types. 
+    - `EXTERNAL_ID` - This identifies the user you want to update. This should match the `external_id` value used in Braze. 
+    - `ALIAS_NAME` and `ALIAS_LABEL` - These two columns create a user alias object. `alias_name` should be a unique identifier, and `alias_label` specifies the type of alias. Users may have multiple aliases with different labels but only one `alias_name` per `alias_label`.
+    - `BRAZE_ID` - The Braze user identifier. This is generated by the Braze SDK, and new users cannot be created using a Braze ID through Cloud Data Ingestion. To create new users, specify an external user ID or user alias. 
+- `PAYLOAD` - This is a string or struct of the fields you want to sync to the user in Braze.
+
+#### Step 2: Create a Access Token  
+
+In order for Braze to access databricks, a personal access token needs to be created.
```
Comment (colleenkerr) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:272]: Databricks

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 2]
```diff
@@ -241,6 +241,62 @@ If you have network policies in place, you must give Braze network access to you
...
+
+1. In your Databricks workspace, click your Databricks username in the top bar, and then select `User Settings` from the drop down.
+2. On the Access tokens tab, click Generate new token.
+3. Enter a comment that helps you to identify this token such as Braze CDI, and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).
+4. Click Generate.
+5. Copy the displayed token, and then click Done.
+
+Keep the token in a safe place until you need to entry it on the Braze dashboard during integration credintial creation.
```
Comment (colleenkerr) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:280]:
```suggestion
Keep the token in a safe place until you need to enter it on the Braze dashboard during the credential creation step.
```

**Review by colleenkerr — CHANGES_REQUESTED** [review]
made a couple suggestions but otherwise looks good 👍

**Review by colleenkerr — CHANGES_REQUESTED** [review]
One more thing - need to add a summary section under "product setup"

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 3]
```diff
@@ -241,6 +247,62 @@ If you have network policies in place, you must give Braze network access to you
...
+    - `EXTERNAL_ID` - This identifies the user you want to update. This should match the `external_id` value used in Braze. 
+    - `ALIAS_NAME` and `ALIAS_LABEL` - These two columns create a user alias object. `alias_name` should be a unique identifier, and `alias_label` specifies the type of alias. Users may have multiple aliases with different labels but only one `alias_name` per `alias_label`.
+    - `BRAZE_ID` - The Braze user identifier. This is generated by the Braze SDK, and new users cannot be created using a Braze ID through Cloud Data Ingestion. To create new users, specify an external user ID or user alias. 
+- `PAYLOAD` - This is a string or struct of the fields you want to sync to the user in Braze.
+
+#### Step 2: Create a Access Token  
+
+In order for Braze to access Databricks, a personal access token needs to be created.
+
+1. In your Databricks workspace, click your Databricks username in the top bar, and then select `User Settings` from the drop down.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:280]:
```suggestion
1. In your Databricks workspace, click your Databricks username in the top bar, and then select **User Settings** from the drop-down.
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 4]
```diff
@@ -241,6 +247,62 @@ If you have network policies in place, you must give Braze network access to you
...
+2. On the Access tokens tab, click Generate new token.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:281]:
```suggestion
2. On the Access tokens tab, click **Generate new token**.
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 5]
```diff
@@ -241,6 +247,62 @@ If you have network policies in place, you must give Braze network access to you
...
+3. Enter a comment that helps you to identify this token such as Braze CDI, and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:282]:
```suggestion
3. Enter a comment that helps you to identify this token, such as "Braze CDI", and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 6]
```diff
@@ -241,6 +247,62 @@ If you have network policies in place, you must give Braze network access to you
...
+4. Click Generate.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:283]:
```suggestion
4. Click **Generate**.
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 7]
```diff
@@ -241,6 +247,62 @@ If you have network policies in place, you must give Braze network access to you
...
+5. Copy the displayed token, and then click Done.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:284]:
```suggestion
5. Copy the displayed token, and then click **Done**.
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md [hunk 8]
```diff
@@ -316,6 +378,27 @@ Next, choose a name for your sync and input contact emails. We'll use this conta
...
+Input the information for your Databricks data warehouse and source table, then proceed to the next step.
+
+![]({% image_buster /assets/img/cloud_ingestion/ingestion_16.png %})
+
+#### Step 2: Configure sync details
+Next, choose a name for your sync and input contact emails. We'll use this contact information to notify you of any integration errors (e.g., access to the table was removed unexpectedly).
+
+![]({% image_buster /assets/img/cloud_ingestion/ingestion_12.png %})
+
+You will also choose the data type and sync frequency. Frequency can be anywhere from every 15 minutes to once per month. We'll use the time zone configured in your Braze dashboard to schedule the recurring sync. Supported data types are Custom Attributes, Custom Events, Purchase Events, and User Deletes. The data type for a sync cannot be changed after creation.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md:400]:
```suggestion
You will also choose the data type and sync frequency. Frequency can be anywhere from every 15 minutes to once per month. We'll use the time zone configured in your Braze dashboard to schedule the recurring sync. Supported data types are custom attributes, custom events, purchase events, and user deletes. The data type for a sync cannot be changed after creation. 
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Edits for formatting standards

**Review by bre-fitzgerald — APPROVED** [review]

**Review by colleenkerr — APPROVED** [review]

**Pull request merged**